app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\CreateUpdateProjectRequest;
+use App\Http\Resources\V1\ProjectResource;
+use App\Models\Project;
+use App\Services\ProjectService;
+
+class ProjectController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return ProjectResource::collection(
+            Project::withCount('tasks')
+                   ->filterByStatus()
+                   ->filterAssignedToUser()
+                   ->orderByDesc('id')
+                   ->paginate(Project::PAGINATE)
```

Comment:
On model you can set `protected $perPage = 25;` and then you won't need const

Reply:
Thank you for this advice!